packages/conf-liburing/conf-liburing.0.1.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Virtual package for system installation of liburing"
+maintainer: ["Lee Koon Wen"]
+authors: ["Lee Koon Wen"]
+license: ["ISC" "BSD-3-Clause"]
+homepage: "https://github.com/koonwen/uring-trace"
+doc: "https://koonwen.github.com/koonwen/uring-trace"
+bug-reports: "https://github.com/koonwen/uring-trace/issues"
+available: [ os = "linux" ]
+
+depexts: [
+  ["liburing-dev"]   { os-distribution = "ubuntu" }
+  ["liburing-dev"]   { os-distribution = "debian" }
+  ["liburing-devel"] { os-distribution = "fedora" }
+]
+flags: conf
+url {
+  src:
+    "https://github.com/koonwen/uring-trace/releases/download/v0.1.0/uring-trace-0.1.0.tbz"
+  checksum: [
+    "sha256=d1469550363df1af0fcb331ac86f6347fb021dfa844532c370317d3ad30d1e91"
+    "sha512=73a519f681d60aa83c4cba8439fc94fe207b0449ef0bff1fef0e976c4f68d6e56f543444cab936910d3e00cb9436abe2828739cc1e27c28d2e8c9d1e9218b1e8"
+  ]
```

Comment:
Afaiu, `conf-` are not meant to include sources, but just contain the depexts. That is why this lint check is failing in our CI:

```
error 46: Package is flagged "conf" but has source, install or remove instructions
```

Is this source actually needed here?